content/docs/1.7.0/snapshots-and-backups/scheduling-backups-and-snapshots.md

```diff
@@ -14,6 +14,8 @@ You can configure,
 - The number of backups or snapshots to retain
 - The number of jobs to run concurrently
 - Any labels that should be applied to the backup or snapshot
+- Parameters that should be applied to the backup or snapshot
+  - `full-backup-interval`: Integer parameter that applied only to the backup. With this parameter, Longhorn performs full backup every N incremental backups. For full backup, see [Create a Backup](../backup-and-restore/create-a-backup)
```

Comment:
This is an important feature for data protection. To effectively promote the feature, we can provide more comprehensive information, including:

- The purpose of the feature.
- Why and when users should enable the feature.
- The benefits of periodic full backups, such as enhanced data protection.
- The drawbacks of periodic full backups, such as increased backup time and cost.

cc @innobead